api/api_endpoints.py

```diff
@@ -0,0 +1,72 @@
+from flask import Blueprint, jsonify, request
+from app import db
+from app.base.models import User
+from api.schema import user_schema, users_schema
+
+api_blueprint = Blueprint("api_blueprint", __name__)
+
+
+@api_blueprint.route('/api/user/all')
```

Comment:
Following CRUD convention you might want to rename this endpoint from
`@api_blueprint.route('/api/user/all')`
to 
`@api_blueprint.route('/api/user/list')`